src/TwigYard/Middleware/Form/Handler/EmailHandler.php

```diff
@@ -55,11 +55,25 @@ public function handle(array $formData)
         $templating = $this->templatingFactory->createTemplating($this->appState);
         $localeSubDir = $this->appState->isSingleLanguage() ? '' : $this->appState->getLocale() . '/';
 
+        try {
```

Comment:
Could we please create a private method that would render the templates. We could then reuse the same code for both subject and body.